src/main/java/seedu/contax/commons/core/Messages.java

```diff
@@ -19,4 +19,7 @@
     public static final String MESSAGE_INVALID_TIME = "The time provided is invalid!";
 
     public static final String MESSAGE_INVALID_INPUT = "=/ provided is invalid";
+
+    public static final String MESSAGE_INVALID_PRIORITY_INDEX = "Invalid index for editpriority";
```

Comment:
You could have just used `MESSAGE_INVALID_APPOINTMENT_DISPLAYED_INDEX`

Reply:
Thx Will change